contracts/PoaMania.sol

```diff
@@ -476,7 +476,11 @@ contract PoaMania is Ownable {
      * @param _roundDuration The round duration (in seconds)
      */
     function _setRoundDuration(uint256 _roundDuration) internal {
-        require(_roundDuration > 0, "should be greater than 0");
+        uint256 randomUpdateInterval = random.getUpdateInterval();
+        require(
+            _roundDuration > randomUpdateInterval.mul(4).mul(blockTime) && _roundDuration > blockTime.mul(40),
```

Comment:
Let's move `randomUpdateInterval.mul(2).mul(blockTime)` to a private function and use such a function in `_setRoundDuration()` and in `getLockStart()`.

How did you get 40 blocks? Why 40 exactly?

Reply:
> Why 40 exactly?

It's just the length of one random collection round. What value do you suggest?

Reply:
Ok, let's maybe change this part of condition to `_roundDuration > blockTime.mul(randomUpdateInterval)` since `randomUpdateInterval` is the collection round length?

Reply:
It's only for the case when `randomUpdateInterval` is 0. But as you said me `randomUpdateInterval` can't be changed, so I'll just remove this check

Reply:
OK